egs2/voxceleb/spk1/run_rep_lrn.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+set -e
+set -u
+set -o pipefail
+
+
+spk_config=conf/tuning/train_rawnet3_decoder_spkonly.yaml
+
+train_set="voxceleb12_devs_sp"
```

Comment:
You do not need to add the suffix _sp manually if --speed_perturb_factors is specified. It will be added automatically.